### DISCUSSION TITLE
Filter on parts of a query

### DISCUSSION THREAD

**Discussion by gerwinbrunner** [google/tamperchrome#203] [Q&A]
Hi,

is it possible to add a filer on only parts of a query?

i.e.:

?operationName=WishlistItems

when the full path is 

https://some.domain.com/api/v1/graphql?operationName=WishlistItems&variables=%7B%22hasMarketplace%22%3Afalse%2C%22shouldFetchBasket%22%3Atrue%2C%22outletId%22%3A%22656%22%2C%22limit%22%3A12%7D&extensions=%7B%22pwa%22%3A%7B%22salesLine%22%3A%22Media%22%2C%22country%22%3A%22AT%22%2C%22language%22%3A%22de%22%7D%2C%22persistedQuery%22%3A%7B%22version%22%3A1%2C%22sha256Hash%22%3A%224eb66e4fb5737b639df85ed89a2ce894d598071cbfbb4d4b20fcf99f6d1c3f40%22%7D%7D

**Comment by sirdarckcat** [comment]
There's currently an strict equality check:
https://github.com/google/tamperchrome/blob/1ad8700e85c719b32ece29931695b3f27b2d5b5a/v2/ui/src/interceptor.service.ts#L191

We could do something like `&param` matches on query params, and `=value` matches on query values, so your filter would be:
`&operationName` and `=WishlistItems`. I agree this would be useful.

**Reply by bogzyeet** [reply]
> There's currently an strict equality check:
> 
> https://github.com/google/tamperchrome/blob/1ad8700e85c719b32ece29931695b3f27b2d5b5a/v2/ui/src/interceptor.service.ts#L191
> 
> We could do something like `&param` matches on query params, and `=value` matches on query values, so your filter would be: `&operationName` and `=WishlistItems`. I agree this would be useful.

and when would you release it lol. Still waiting!
